### DISCUSSION TITLE
ios WeChat  use await fp.get()  fail

### DISCUSSION THREAD

**Discussion by jyxu1016** [fingerprintjs/fingerprintjs#664] [Q&A]
ios WeChat  use await fp.get()  fail

**Comment by Finesse** [comment]
What iOS, WeChat and FingerprintJS versions do you use?

What's the error message?

Please provide a reproducible demo (a HTML page).

**Reply by jyxu1016** [reply]
// js: 
import FingerprintJS from '@fingerprintjs/fingerprintjs'
const fpPromise = FingerprintJS.load();
(async () => {
  const fp = await fpPromise
  const result = await fp.get()
  const visitorId = result.visitorId
  console.log(visitorId)
})()

// fingerprintjs versions  :  "@fingerprintjs/fingerprintjs": "^3.1.2",

Apple browser can open normally，But through wechat QR code to enter white screen

**Reply by Finesse** [reply]
What's the iOS and WeChat versions?

**Reply by jyxu1016** [reply]
ios: 12.3.1
WeChat : 8.0.6

**Reply by Finesse** [reply]
I've moved the conversation to https://github.com/fingerprintjs/fingerprintjs/issues/645#issuecomment-851895669. Please leave your future comments regarding the bug there.
